app/styles/list-view.scss

```diff
@@ -1,5 +1,6 @@
 .list-view {
   width: 100%;
+  padding: 0 140px;
```

Comment:
You introduce $spacer-layout (130px) but still use a constant here - seems inconsistent?